app/serializers/responsibility_index_serializer.rb

```diff
@@ -1,5 +1,6 @@
 class ResponsibilityIndexSerializer < ResponsibilityBaseSerializer
 
   has_one :student, serializer: StudentBaseSerializer
+  has_one :user, serializer: UserBaseSerializer
```

Comment:
We actually can't do this because we're going to get cyclical problems with serializers (this serializer has_one :user, and then that `:user` has many `:responsibilities` - see how that cycles?).

What is using this `:user` right now?